src/main/java/com/google/api/codegen/transformer/StaticLangApiMethodTransformer.java

```diff
@@ -324,7 +334,7 @@ public StaticLangApiMethodView generateRequestObjectAsyncMethod(
         Synchronicity.Async,
         additionalParams,
         methodViewBuilder,
-        Arrays.asList(type));
+        Arrays.asList(CallingForm.RequestAsync));
```

Comment:
How do we know this is always `RequestAsync`?

Reply:
We're in method `generateRequestObjectAsyncMethod`, so I'm taking the `Request` and `Async` at face value. Is this incorrect? If so, we can trace the logic more thoroughly.

Reply:
I looked through the code a little more. C# seems to also use the `type` for cancellation method. I'm good with this PR, but we might need to revisit.